### PR TITLE
Give better names to test cases.

### PR DESCRIPTION
**Pull request by platfowner**
Summary:
- Rename test cases
- Move sharding-related native function tests to sharding.test.js

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,12 +16,15 @@
...
-    "test_smoke": "./node_modules/mocha/bin/mocha --timeout 160000 --exclude integration/integration.test.js \"integration/**/*.test.js\"",
-    "loadtest": "sh loadtest/load_tester.sh",
-    "test_db": "mocha ./test/db.test.js",
-    "test_state_util": "mocha ./test/state-util.test.js"
+    "test_state_util": "./node_modules/mocha/bin/mocha --timeout 160000 unittest/state-util.test.js",
+    "test_db": "./node_modules/mocha/bin/mocha --timeout 160000 unittest/db.test.js",
+    "test_unit": "NUM_VALIDATORS=1 ./node_modules/mocha/bin/mocha --timeout 160000 \"unittest/**/*.test.js\"",
+    "test_node": "./node_modules/mocha/bin/mocha --timeout 160000 integration/node.test.js",
+    "test_blockchain": "./node_modules/mocha/bin/mocha --timeout 160000 integration/blockchain.test.js",
+    "test_afan": "./node_modules/mocha/bin/mocha --timeout 160000 integration/afan.test.js",
```
Comment (liayoo) [inline package.json:24]: How about we change this to a more general name? e.g. test_app or test_dapp

**Review by liayoo — APPROVED** [review]
LGTM with a minor comment. Thanks for giving clearer names to the tests!

File: package.json (hunk 1 above)
Comment (platfowner) [reply package.json]: Thanks for the suggestion. Just renamed to 'test_dapp' with 'afan_dapp.test.js' filename.

**Comment by platfowner** [comment]
Comments addressed and, as an additional change just moved sharding-related native function tests to sharding.test.js.
Let me know if you have any comments on that. @liayoo

**Comment by liayoo** [comment]
@platfowner Moving to sharding.test.js sounds good to me. Thanks!

**Comment by platfowner** [comment]
Thanks for the review! Now merging..

**Pull request merged**